code/modules/surgery/organs/parasites.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/organ/internal/body_egg/spider_eggs
+	name = "spider eggs"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "eggs"
+	var/stage = 1
+
+/obj/item/organ/internal/body_egg/spider_eggs/on_life()
+	if(stage < 5 && prob(3))
```

Comment:
It's not even the base type that does it though; it's a subtype of a basetype that does it, and primarily so that specific egg (xenomorph embryo) can continue to grow, even outside of a host (and technically inside, even if dead).

This also has the advantage of never falling out of sync--should the mob's be lagging, then this will lag, too---where as if object processing is lagging (or not), then it'll separate out, on the ticks, a bit more than it otherwise would; I'd tend to think it more ideal for as much stuff to be processing on that particular single mob, when that mob is processed, than be based on another tick when that tick processes.